README.md

```diff
@@ -1,64 +1,88 @@
-<p align="center">
-<img src="https://github.com/CesiumGS/cesium/wiki/logos/Cesium_Logo_Color.jpg" width="50%" />
-</p>
+# CesiumJS
+
+![Cesium](https://github.com/CesiumGS/cesium/wiki/logos/Cesium_Logo_Color.jpg)
```

Comment:
Feel free to disregard this as it is purely aesthetic, but would it be possible to not use the white background here? It's always so bright and out of place on dark mode. Now that GitHub has dark mode support, [we can include different images for light and dark](https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown/).

Reply:
Great idea!

Reply:
I'm not sure SVGs are supported in GitHub Markdown. I'm seeing blank spots on web and the GitHub mobile app.

Reply:
Interesting, I'm following the best practices outlines for images in the [official docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images). Also the docs use an SVG in the example so my guess is they are supported, at least officially. 

If we're seeding problems, I'll restore the previous logo with background.

Reply:
Yeah, that's really strange that it's not working. I wonder if using PNGs with transparency instead of SVGs would work.

Reply:
Given the issue on mobile, I'm going to rollback to the old banner image for now.